### PR TITLE
Fix/address form

### PR DESCRIPTION
**Pull request by RoeiRom**
# Address Form Refactor
## The Problem
Now the addresses in the investigation form are not ordered by the same order throughout the system
## The solution
* Creating a generic component that is placed anywhere there is a use of an address in the investigation form
> Refactoring the personal information tab to use mostly with react-hook-form (🤢) (Will be in anouther MR)

### CODE REVIEW

File: client/src/components/App/Content/InvestigationForm/TabManagement/InteractionsTab/InteractionDialog/InteractionEventForm/InteractionSection/InteractionEventForm.tsx [hunk 1]
```diff
@@ -1,29 +1,41 @@
...
+
+const ADDRESS_LABEL = 'כתובת';
 
 const InteractionEventForm: React.FC<InteractionEventFormProps> = (
     { onPlaceSubTypeChange, isVisible, interactionData, isNewInteraction }: InteractionEventFormProps): JSX.Element => {
+    
     const {control, watch, clearErrors, setValue, errors, setError} = useFormContext();
+    const patientAddress = useSelector<StoreStateType, FlattenedDBAddress>(state => state.address);
+    
+
```
Comment (doronlnd84) [inline client/src/components/App/Content/InvestigationForm/TabManagement/InteractionsTab/InteractionDialog/InteractionEventForm/InteractionSection/InteractionEventForm.tsx:37]: remove row :)

File: client/src/components/App/Content/InvestigationForm/TabManagement/InteractionsTab/InteractionDialog/InteractionEventForm/InteractionSection/InteractionEventForm.tsx [hunk 2]
```diff
@@ -1,29 +1,41 @@
...
-import PatientAddress from './PatientAddress';
-import AddressForm from './AddressForm/AddressForm';
 import useStyles from './InteractionEventFormStyles';
+import GoogleAddressForm from './AddressForm/AddressForm';
 import PlaceNameForm from './PlaceNameForm/PlaceNameForm';
-import BusinessContactForm from './BusinessContactForm/BusinessContactForm';
 import useContactEvent from 'Utils/ContactEvent/useContactEvent';
+import FormRowWithInput from 'commons/FormRowWithInput/FormRowWithInput';
+import BusinessContactForm from './BusinessContactForm/BusinessContactForm';
+
```
Comment (doronlnd84) [inline client/src/components/App/Content/InvestigationForm/TabManagement/InteractionsTab/InteractionDialog/InteractionEventForm/InteractionSection/InteractionEventForm.tsx:27]: remove row :)

File: client/src/components/App/Content/InvestigationForm/TabManagement/InteractionsTab/InteractionDialog/InteractionEventForm/InteractionSection/InteractionEventForm.tsx (hunk 1 above)
Comment (RoeiRom) [reply client/src/components/App/Content/InvestigationForm/TabManagement/InteractionsTab/InteractionDialog/InteractionEventForm/InteractionSection/InteractionEventForm.tsx]: Done 😀

File: client/src/components/App/Content/InvestigationForm/TabManagement/InteractionsTab/InteractionDialog/InteractionEventForm/InteractionSection/InteractionEventForm.tsx (hunk 2 above)
Comment (RoeiRom) [reply client/src/components/App/Content/InvestigationForm/TabManagement/InteractionsTab/InteractionDialog/InteractionEventForm/InteractionSection/InteractionEventForm.tsx:27]: Done 😀

**Review by doronlnd84 — DISMISSED** [review]

File: client/src/components/App/Content/InvestigationForm/TabManagement/PersonalInfoTab/PersonalInfoTab.tsx [hunk 3]
```diff
@@ -456,36 +456,36 @@ const PersonalInfoTab: React.FC<Props> = ({ id }: Props): JSX.Element => {
...
+                                            testId='personalDetailsHouseNumber'
+                                            name={PersonalInfoDataContextFields.HOUSE_NUMBER}
                                             value={props.value}
                                             onChange={(newValue: string) => props.onChange(newValue)}
                                             onBlur={props.onBlur}
-                                            placeholder='קומה'
-                                            label='קומה'
-                                            className={classes.floorInput}
+                                            placeholder='מספר בית'
+                                            label='מספר בית'
```
Comment (Noam6421) [inline client/src/components/App/Content/InvestigationForm/TabManagement/PersonalInfoTab/PersonalInfoTab.tsx:469]: maybe make a const?

File: client/src/components/App/Content/InvestigationForm/TabManagement/PersonalInfoTab/PersonalInfoTab.tsx [hunk 4]
```diff
@@ -456,36 +456,36 @@ const PersonalInfoTab: React.FC<Props> = ({ id }: Props): JSX.Element => {
...
+                                            testId='personalDetailsFloor'
+                                            name={PersonalInfoDataContextFields.FLOOR}
                                             value={props.value}
                                             onChange={(newValue: string) => props.onChange(newValue)}
                                             onBlur={props.onBlur}
-                                            placeholder='מספר בית'
-                                            label='מספר בית'
-                                            className={classes.houseNumInput}
+                                            placeholder='קומה'
+                                            label='קומה'
```
Comment (Noam6421) [inline client/src/components/App/Content/InvestigationForm/TabManagement/PersonalInfoTab/PersonalInfoTab.tsx:487]: same

**Review by MayaSasson — APPROVED** [review]

**Review by Noam6421 — APPROVED** [review]

**Pull request merged**